AmplifyPlugins/DataStore/Tests/DataStoreHostApp/DataStoreStressTests/README.md

```diff
@@ -0,0 +1,250 @@
+
+type Post5 @model {
+  id: ID!
+  title: String!
+  editors: [PostEditor5] @connection(keyName: "byPost5", fields: ["id"])
+}
+
+# Create a join model
+type PostEditor5
+  @model
+  @key(name: "byPost5", fields: ["postID", "editorID"])
+  @key(name: "byEditor5", fields: ["editorID", "postID"]) {
+  id: ID!
+  postID: ID!
+  editorID: ID!
+  post: Post5! @connection(fields: ["postID"])
+  editor: User5! @connection(fields: ["editorID"])
+}
+
+type User5 @model {
+  id: ID!
+  username: String!
+  posts: [PostEditor5] @connection(keyName: "byEditor5", fields: ["id"])
+}
+
+# This is one of the default schemas provided when you run `amplify add api`
+# > Do you have an annotated GraphQL schema? `No`
+# > Choose a schema template: `One-to-many relationship (e.g., “Blogs” with “Posts” and “Comments”)`
+
+# 6 - Blog Post Comment
+type Blog6 @model {
+  id: ID!
+  name: String!
+  posts: [Post6] @connection(keyName: "byBlog", fields: ["id"])
+}
+
+type Post6 @model @key(name: "byBlog", fields: ["blogID"]) {
+  id: ID!
+  title: String!
+  blogID: ID!
+  blog: Blog6 @connection(fields: ["blogID"])
+  comments: [Comment6] @connection(keyName: "byPost", fields: ["id"])
+}
+
+type Comment6 @model @key(name: "byPost", fields: ["postID", "content"]) {
+  id: ID!
+  postID: ID!
+  post: Post6 @connection(fields: ["postID"])
+  content: String!
+}
+
+# Scalars, Lists, Enums
+
+type ScalarContainer @model {
+   id: ID!
+   myString: String
+   myInt: Int
+   myDouble: Float
+   myBool: Boolean
+   myDate: AWSDate
+   myTime: AWSTime
+   myDateTime: AWSDateTime
+   myTimeStamp: AWSTimestamp
+   myEmail: AWSEmail
+   myJSON: AWSJSON
+   myPhone: AWSPhone
+   myURL: AWSURL
+   myIPAddress: AWSIPAddress
+}
+
+type ListIntContainer @model {
+  id: ID!
+  test: Int!
+  nullableInt: Int
+  intList: [Int!]!
+  intNullableList: [Int!]
+  nullableIntList: [Int]!
+  nullableIntNullableList: [Int]
+}
+
+type ListStringContainer @model {
+  id: ID!
+  test: String!
+  nullableString: String
+  stringList: [String!]!
+  stringNullableList: [String!]
+  nullableStringList: [String]!
+  nullableStringNullableList: [String]
+}
+
+type EnumTestModel @model {
+  id: ID!
+  enumVal: TestEnum!
+  nullableEnumVal: TestEnum
+  enumList: [TestEnum!]!
+  enumNullableList: [TestEnum!]
+  nullableEnumList: [TestEnum]!
+  nullableEnumNullableList: [TestEnum]
+}
+
+enum TestEnum {
+  VALUE_ONE
+  VALUE_TWO
+}
+
+type NestedTypeTestModel @model {
+  id: ID!
+  nestedVal: Nested!
+  nullableNestedVal: Nested
+  nestedList: [Nested!]!
+  nestedNullableList: [Nested!]
+  nullableNestedList: [Nested]!
+  nullableNestedNullableList: [Nested]
+}
+
+type Nested {
+  valueOne: Int
+  valueTwo: String
+}
+
+type CustomerOrder @model
+   @key(fields: ["orderId","id"]) {
+   id: ID!
+   orderId: String!
+   email: String!
+}
+
+```
+3. If you are using the latest CLI, update cli.json to include `"useExperimentalPipelinedTransformer": false` to ensure that it will use the v1 transformer.
```

Comment:
we should be using the latest defaults for new tests we are writing, ie. TransformerV2

Reply:
which means we should also use V2 schema for testing, migrate `@connection` usage to `@hasMany`/`@belongsTo` etc